pkg/common/util.go

```diff
@@ -0,0 +1,22 @@
+package common
+
+import (
+	"crypto/sha256"
+	"encoding/binary"
+	"math/rand"
+)
+
+// RandomString generates a hash string with fixed length from "a-zA-Z0-9" using
+// a seed string and provided length.
+func RandomString(seed string, length int) string {
+	h := sha256.New()
+	h.Write([]byte(seed))
+	rand.Seed(int64(binary.BigEndian.Uint64(h.Sum(nil))))
```

Comment:
The function name is a bit misleading since this isn't a random function, the output is deterministic based on the input string.

Is the goal to just hash the input string into a consistent length? Elsewhere in the code we use [md5 sums to make strings like that](https://github.com/getporter/porter/blob/784d61cf283bbd5b9c05a42ea718d3d042803305/pkg/cache/cached_bundle.go#L49-L54). I think that would be a bit more straightforward if what you are looking for is a hash. What do you think?